services/order.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from typing import Optional
+
+from db.models import Order, Ticket
+
+
+def create_order(
+    tickets: list[dict[str]],
+    username: str,
+    date: Optional[str] = None
+) -> Order:
+
+    with transaction.atomic():

```

Comment:
Use ` transaction.atomic` as decorator instead of context manager

Reply:
Thanks!